### ISSUE TITLE
Adding `workbench.editors.webviewInput` to `workbench.editor.experimentalAutoLockGroups` is not useful

### ISSUE THREAD

**Issue by aeschli** [microsoft/vscode#131536]
Testing #131196

Add `workbench.editors.webviewInput` to `workbench.editor.experimentalAutoLockGroups`

```
	"workbench.editor.experimentalAutoLockGroups": [
		"workbench.editors.terminal", "workbench.editors.webviewInput"
	]
```

Open a markdown file and open a preview to the side.
The preview is locked, it uses an editor of type `workbench.editors.webviewInput`

As a user I don't know/care what editor type is used for a view. It's an implementation detail that the preview uses a webview.
As a user I want a setting that says 'markdownPreview` or preview.

**Comment by ArturoDent** [comment]
I have no idea what entries are allowed in the `workbench.editor.experimentalAutoLockGroups` settings input.

**Comment by bpasero** [comment]
Yes, we need to find a way to allow either the editor type or specific view types of that editor.

> I have no idea what entries are allowed in the workbench.editor.experimentalAutoLockGroups settings input.

We have https://github.com/microsoft/vscode/issues/131587 to investigate a solution to that.
